### ISSUE TITLE
Fixed CricleSnail background Color fill transparent

### ISSUE THREAD

**Issue by jayeshpurohit12** [oblador/react-native-progress#264]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `react-native-progress@5.0.0` for the project I'm working on.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/react-native-progress/Circle.js b/node_modules/react-native-progress/Circle.js
index 79f0a9e..251b5e4 100644
--- a/node_modules/react-native-progress/Circle.js
+++ b/node_modules/react-native-progress/Circle.js
@@ -136,6 +136,7 @@ export class ProgressCircle extends Component {
                 }
               : undefined
           }
+          color={fill}
         >
           {unfilledColor && progressValue !== 1 ? (
             <Shape
@@ -174,6 +175,7 @@ export class ProgressCircle extends Component {
               stroke={borderColor || color}
               strokeCap={strokeCap}
               strokeWidth={border}
+              fill='transparent'
             />
           ) : (
             false
diff --git a/node_modules/react-native-progress/CircleSnail.js b/node_modules/react-native-progress/CircleSnail.js
index cdf2b2d..d55e500 100644
--- a/node_modules/react-native-progress/CircleSnail.js
+++ b/node_modules/react-native-progress/CircleSnail.js
@@ -27,6 +27,7 @@ export default class CircleSnail extends Component {
     thickness: PropTypes.number,
     strokeCap: PropTypes.string,
     useNativeDriver: PropTypes.bool,
+    fill: PropTypes.string,
   };
 
   static defaultProps = {
@@ -38,6 +39,7 @@ export default class CircleSnail extends Component {
     thickness: 3,
     strokeCap: 'round',
     useNativeDriver: false,
+    fill: 'transparent',
   };
 
   constructor(props) {
@@ -85,7 +87,7 @@ export default class CircleSnail extends Component {
         easing: Easing.inOut(Easing.quad),
         useNativeDriver: this.props.useNativeDriver,
       }),
-    ]).start(endState => {
+    ]).start((endState) => {
       if (endState.finished) {
         if (Array.isArray(this.props.color)) {
           this.setState({
@@ -104,7 +106,7 @@ export default class CircleSnail extends Component {
       easing: Easing.linear,
       isInteraction: false,
       useNativeDriver: this.props.useNativeDriver,
-    }).start(endState => {
+    }).start((endState) => {
       if (endState.finished) {
         this.state.rotation.setValue(0);
         this.spin();
@@ -129,6 +131,7 @@ export default class CircleSnail extends Component {
       style,
       thickness,
       strokeCap,
+      fill,
       ...restProps
     } = this.props;
 
@@ -163,7 +166,7 @@ export default class CircleSnail extends Component {
           },
         ]}
       >
-        <Svg width={size} height={size}>
+        <Svg width={size} height={size} fill={fill}>
           <AnimatedArc
             direction={
               direction === 'counter-clockwise'
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by ebulku** [comment]
The easiest way is to change the version of react-native-progress with the PR# fixing this issue in package.json file and run `npm update`

 `"react-native-progress": "oblador/react-native-progress#pull/258/head"`

**Comment by draperunner** [comment]
I had the same (or similar) problem with the fill of `Circle` suddenly being black instead of transparent by default. I simply added the prop `fill="transparent"` to my usage of the `<Circle />` component.

I guess an easier fix for the maintainer would be setting `fill: "transparent"` in the defaultProps. 

<img width="62" alt="bilde" src="https://github.com/oblador/react-native-progress/assets/4339443/48f9e8b6-c6b5-446f-b2b6-ccca6265e72c">
➡️ 
<img width="62" alt="bilde" src="https://github.com/oblador/react-native-progress/assets/4339443/e99641ff-a608-4665-bd45-4d4cb23103ca">

**Comment by LucasHimelfarb** [comment]
@draperunner eye, I need change the color of the circle indicator. How you do that?

**Comment by oblador** [comment]
Should be fixed in [5.0.1](https://github.com/oblador/react-native-progress/releases/tag/v5.0.1)
